src/openloco/windows/options.cpp

```diff
@@ -2125,6 +2110,12 @@ namespace openloco::ui::options
         controls::init_events();
         misc::init_events();
 
+        // Update config fullscreen resolution if not set
+        auto& cfg = config::get();
+        auto& cfg_new = config::get_new();
+        if (!(cfg_new.display.fullscreen_resolution.width > 0 && cfg_new.display.fullscreen_resolution.height > 0 && cfg.resolution_width > 0 && cfg.resolution_height > 0))
+            updateFullscreenResolutions();
```

Comment:
The reason I put this here was so that the `config.display.fullscreen_resolution.width/height` variables were set to the last/highest available resolution if they were not already configured.

I wanted to make sure this occurred before the Options window was drawn otherwise if they haven't been previously configured, the "Display Resolution" dropdown shows as "0x0".

I could use a less involved function and just grab the **current** screen resolution and show that if one hasn't been previously configured.

Reply:
I've removed it for now. The "Options" window shows "0x0" as the resolution the first time you open it, but as soon as you switch to "Fullscreen" it switches to the current desktop resolution and fills in the dropdown menu.